src/Model/Resolver/OrderListResolver.php

```diff
@@ -28,23 +29,34 @@ class OrderListResolver implements ResolverInterface
     /**
      * @var CollectionFactoryInterface
      */
-    private $collectionFactory;
+    protected $collectionFactory;
 
     /**
      * @var CheckCustomerAccount
      */
-    private $checkCustomerAccount;
+    protected $checkCustomerAccount;
+
+
+    /**
+     * @var CustomerRepositoryInterface
+     */
+    protected $customerRepository;
 
     /**
      * @param CollectionFactoryInterface $collectionFactory
      * @param CheckCustomerAccount $checkCustomerAccount
+     * @param CustomerRepositoryInterface $customerRepository
      */
     public function __construct(
         CollectionFactoryInterface $collectionFactory,
-        CheckCustomerAccount $checkCustomerAccount
+        CheckCustomerAccount $checkCustomerAccount,
+        CustomerRepositoryInterface $customerRepository
     ) {
         $this->collectionFactory = $collectionFactory;
         $this->checkCustomerAccount = $checkCustomerAccount;
+        $this->customerRepository = $customerRepository;
+
+        parent::__construct($collectionFactory, $checkCustomerAccount);
```

Comment:
Do we really have to call parent constructor method when we don't extend anything?

Reply:
I find it good practice to always call parent as I have had cases where if parent is not called the Magento inheritance system will not be fully happy and something will not work, so as a rule of thumb I always call / intit parent as well